src/Package/Impl/DataInspect/DataImport/ImportDataWindow.xaml.cs

```diff
@@ -162,6 +164,12 @@ public ImportDataWindow(string filePath, string name)
         }
 
         private async Task PreviewAsync() {
+            if(_encodingsInitTask == null) {
```

Comment:
It is null when InitializeComponent creates combobox and it fires selection change